css/style.css

```diff
@@ -0,0 +1,248 @@
+body{
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+    display: block;
+
+}
+
+.container{
+    
+    background-color: #262647;
+    margin-top: 3px;
+    margin-bottom: 3px;
+    margin-right: 300px;
+    margin-left: 300px;
+    padding: 16px;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+:root{
+    --color-white: #f3f3f3;
+    --color-darkblue: #1b1b32;
+    --color-darkblue-alpha: rgba(27,27,50,0.8);
+    --color-green: #37af65;
+}
+
+body::before{
```

Comment:
Remove unnecessary space after the end of the style. This makes your code not clean and easy to read by other programmers.
It's not a CSS best practice